tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -205,6 +205,40 @@ public function getSpaceByName(string $user, string $spaceName): array {
 		return $spaces[$spaceName];
 	}
 
+	/**
+	 * The method returns userId
+	 *
+	 * @param string $userName
+	 *
+	 * @return string
+	 */
+	public function getUserIdByUserName(string $userName): string {
+
+		$fullUrl = $this->baseUrl . "/api/v0/accounts/accounts-list";
+		$this->featureContext->setResponse(
+			HttpRequestHelper::post(
+				$fullUrl, 
+				"", 
+				$this->featureContext->getAdminUsername(), 
+				$this->featureContext->getAdminPassword(), 
+				[], 
+				"{}"
+			)
+		);
+		if ($this->featureContext->getResponse()) {
+			$rawBody = $this->featureContext->getResponse()->getBody()->getContents();
+			if (isset(\json_decode($rawBody, true, 512, JSON_THROW_ON_ERROR)["accounts"])) {
+				$accounts = \json_decode($rawBody, true, 512, JSON_THROW_ON_ERROR)["accounts"];
+			}
+		}
```

Comment:
This could have an `else` that throws a specific error telling the tester/developer that the accounts-list response was empty. It is helpful to give the best clues we can about why a test step fails.